tools/harnesses/gen_fuzz_sqlquery_corpus.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
```

Comment:
Mind replacing this with
```
#!/usr/bin/env bash

#  Copyright (c) 2014-present, Facebook, Inc.
#  All rights reserved.
#
#  This source code is licensed in accordance with the terms specified in
#  the LICENSE file found in the root directory of this source tree.
```

Reply:
I suggest naming this `osqueryfuzz_sqlquery_corpus.sh` to conform with the harness binary+dict naming.